shared/js/ui/templates/grade-scorecard.es6.js

```diff
@@ -0,0 +1,65 @@
+const bel = require('bel')
+const siteRating = require('./shared/site-rating.es6.js')
+const siteRatingSubtitle = require('./shared/site-rating-subtitle.es6.js')
+
+module.exports = function () {
+  return bel`<section class="sliding-subview sliding-subview--has-fixed-header">
+   <div class="site-info site-info--full-height card">
+     <div class="hero border--bottom">
+      <a href="#" class="hero__close js-sliding-subview-close">
+        <span class="icon icon__arrow icon__arrow--large icon__arrow--left">
+        </span>
+      </a>
+      ${siteRating(
+        this.model.site.isCalculatingSiteRating,
+        this.model.site.siteRating,
+        this.model.site.isWhitelisted
+      )}
+      <h1 class="hero__title">${this.model.site.domain}</h1>
+      ${siteRatingSubtitle(
+        this.model.site.isCalculatingSiteRating,
+        this.model.site.siteRating,
+        this.model.site.isWhitelisted
+      )}
+    </div>
+    ${getReasons(this.model.reasons)}
+    ${getGrades(this.model.site.siteRating)}
+  </div>
+  </section>`
+}
+
+function getReasons (reasons) {
+  if (!reasons.length) { return }
+
+  return bel`<ul class="status-list status-list--right padded border--bottom--inner">
+    ${reasons.map(item => { return renderItem(item.modifier, item.msg) })}
```

Comment:
es6 syntax would be:
`${reasons.map(item => renderItem(item.modifier, item.msg) )}`

The curly braces and `return` are implied in this form.